libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -538,7 +538,7 @@ protected void onPostExecute(OAuth2.TokenEndpointResponse tr) {
                 final PasscodeManager passcodeManager = mgr.getPasscodeManager();
                 passcodeManager.storeMobilePolicyForOrg(account, id.screenLockTimeout * 1000 * 60, id.pinLength);
                 passcodeManager.setTimeoutMs(id.screenLockTimeout * 1000 * 60);
-                passcodeManager.setMinPasscodeLength(id.pinLength);
+                boolean changeRequired = passcodeManager.setMinPasscodeLength((Activity) getContext(), id.pinLength);
```

Comment:
We need to pass in the Activity context here, so that LoginActivity is used, and so that LoginActivity can get onActivityResult from the PasscodeActivity. Without this, the application context is used, and we don't get a callback when the passcode is done.